packages/volto-slate/src/blocks/Text/index.js

```diff
@@ -118,6 +121,27 @@ export default function applyConfig(config) {
       addPermission: [],
       view: [],
     },
+    cloneData: (data) => {
+      return [
+        uuid(),
+        {
+          ...data,
+          value: [
+            ...(data?.value || []).map((c) => {
+              return {
+                ...c,
+                children: (c?.children || []).map((childrenData) => {
```

Comment:
Won't this add `children` to nodes that should not have it, like text nodes?

I would really like to see a test for this feature.

Reply:
Yes this is true, I will modify, and add tests